resource/reapi/bindings/c/reapi_cli.h

```diff
@@ -67,6 +100,22 @@ int reapi_cli_match_allocate (reapi_cli_ctx_t *ctx, bool orelse_reserve,
                               bool *reserved,
                               char **R, int64_t *at, double *ov);
 
+/*! Match a jobspec to the "best" resources and either allocate
+ *  orelse reserve them. The best resources are determined by
+ *  the selected match policy.
```

Comment:
Typo in description:
```suggestion
/*! Run Satisfiability check for jobspec. 
```